composer.json

```diff
@@ -12,15 +12,19 @@
 	],
 	"config": {
 		"sort-packages": true,
-		"preferred-install": "dist"
+		"preferred-install": "dist",
+		"allow-plugins": {
+			"composer/installers": true,
+			"dealerdirect/phpcodesniffer-composer-installer": true
+		}
```

Comment:
Do we want to keep these in here? What's our policy?

Reply:
No policy per se but I think this is fine. Probably not strictly required for a dependency but necessary for local dev / linting in actions etc...